src/biblenotify/ui/BibleNotify/Views/HomeView.qml

```diff
@@ -9,38 +9,65 @@ Item {
     id: root
 
     property var setTimeView
+    property string verseText
+    property string verseReference
```

Comment:
Why not just use a single property for verse (like in [main.qml:22](https://github.com/BibleNotify/BibleNotifyDesktop/pull/16/files#diff-dd3914ea29f84f5c2559011aab9d26201d20be4aa3ba9447b1844b3230c18e5bR22))?

(Side note mainly for me: I will change the verse to be a QVariantMap after this PR is merged as `root.verse[2]` etc. is not easy to read/understand)

Reply:
> Why not just use a single property for verse

Primarily because I didn't want to access it with the index (which is confusing) as you said. 

``text: verseReference`` vs. ``text: verse[0] // or [1] - I don't even remember 🙄 ``

I agree that a map would be a much better solution so if you don't mind I suggest we leave this as is.

Reply:
It would give me slightly less work. :wink:

But yes I guess we can leave this in for now.